src/style.css

```diff
@@ -2,0 +1,28 @@
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  align-items: center;
+  box-sizing: border-box;
+  padding-inline-start: 0;
```

Comment:
The `padding-inline-start` property is used for setting the padding at the start of an inline element, which is not necessary here as the `.stars` class is for a flex container. If you want to reset the padding of the list, use `padding-left` instead, considering the direction of the document.